src/crimson/net/SocketConnection.h

```diff
@@ -150,9 +153,17 @@ class SocketConnection : public Connection {
 
   seastar::future<> fault();
 
+  seastar::future<> dispatch();
+
+  /// Only call when SocketConnection first construct
```

Comment:
i think the original comment of

> `/// complete a handshake from the client's perspective`

still hold. probably "handshake" is more about the internals of `start_connect()`, but "from the client's perspective" does help to explain when/how this method is supposed to be used.

Reply:
done.